File: ClickServer/generatedJava/com/pku/ss/egl/prog/PlayMusic.java
package com.pku.ss.egl.prog;
import org.eclipse.edt.javart.resources.*;
import org.eclipse.edt.javart.*;
import eglx.lang.SysLib;
import org.eclipse.edt.runtime.java.eglx.lang.EAny;
import java.lang.Object;
@SuppressWarnings("unused")
@javax.xml.bind.annotation.XmlRootElement(name="PlayMusic")
public class PlayMusic extends ProgramBase {
	private static final long serialVersionUID = 10L;
	
	public static void main(String... ezeargs) throws Exception {
		StartupInfo info = new StartupInfo( "PlayMusic", "com/pku/ss/egl/prog/PlayMusic.properties", ezeargs );
		RunUnit ru = new JSERunUnit( info );
		org.eclipse.edt.javart.Runtime.setStaticRunUnit( ru );
		ru.start( new PlayMusic() );
		ru.exit();
	}
	public PlayMusic() {
		super();
	}
	{
		if(org.eclipse.edt.javart.Runtime.getRunUnit().getActiveExecutable() == null)
		org.eclipse.edt.javart.Runtime.getRunUnit().setActiveExecutable(this);
		ezeInitialize();
	}
	public void ezeInitialize() {
	}
	public void main() {
		SysLib.startCmd("music\\Flourishing.mp3");
	}
}

File: ClickServerWeb/WebContent/com/pku/ss/egl/prog/PlayMusic.js
define([],function(){
	egl.defineClass('com.pku.ss.egl.prog', "PlayMusic", ProgramBase, {
		"eze$$fileName" : "com/pku/ss/egl/prog/PlayMusic.egl",
			,
			"eze$$setEmpty": function() {
			}
			,
			"eze$$setInitial": function() {
				this.eze$$setEmpty();
			}
			,
			"eze$$clone": function() {
				var ezert$$1 = this;
				var ezert$$2 = new egl.com.pku.ss.egl.prog.PlayMusic();
				return ezert$$2;
			}
			,
			"eze$$getAnnotations": function() {
				if(this.annotations === undefined){
					this.annotations = {};
					this.annotations["XMLRootElement"] = new egl.eglx.xml.binding.annotation.XMLRootElement("PlayMusic", null, false);
				}
				return this.annotations;
			}
			,
			"eze$$getFieldInfos": function() {
				if(this.fieldInfos === undefined){
					var annotations;
					this.fieldInfos = new Array();
				}
				return this.fieldInfos;
			}
			,
			"main": function() {
				egl.eglx.lang.SysLib.startCmd("music\\Flourishing.mp3");
			}
			,
			"toString": function() {
				return "[PlayMusic]";
			}
		}
	);
});
